utils/Types.utils.js

```diff
@@ -37,38 +37,40 @@ export function asyncApiToJavaType(type, format) {
 
 function asyncApiFormatToJavaType(format) {
   switch (format) {
-    case 'int32':
-      return 'int';
-    case 'int64':
-      return 'long';
-    case 'float':
-      return 'float';
-    case 'double':
-      return 'double';
-    case 'byte':
-      return 'byte';
-    case 'binary':
-      return 'String';
-    case 'date':
-      return 'String';
-    case 'date-time':
-      return 'String';
-    case 'password':
-      return 'String';
+  case 'int32':
+    return 'int';
+  case 'int64':
+    return 'long';
+  case 'float':
+    return 'float';
+  case 'double':
+    return 'double';
+  case 'byte':
+    return 'byte';
+  case 'binary':
+    return 'String';
+  case 'date':
+    return 'String';
+  case 'date-time':
+    return 'String';
+  case 'password':
+    return 'String';
+  case 'uuid':
```

Comment:
`uuid` isn't a format in the AsyncAPI spec but it's commonly used enough that it seems reasonable to add

We'll need to update the template to add `java.util.UUID` to the imports for any class that uses it, though - as it's not a primitive

Reply:
@dalelane worth importing `java.util.UUID` regardless or doing it conditionally?

Reply:
depends on how much effort it would be, I guess - it'd be nice to not add unused imports, but it wouldn't cause errors so it's not worth a massive amount of work, IMO

Reply:
I've just imported it for now, but I think a future task is to cleanup how we do this, as its likely future datatypes will make this more of an issue